### PR TITLE
First commit!

### PR DESCRIPTION
**Pull request by kesslerb**
### Caption

Change the caption in kesslerb test branch.

### CODE REVIEW

File: _posts/0000-01-07-kesslerb.md [hunk 1]
```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![minertocat](https://octodex.github.com/images/minertocat.png)
 {: .center}
 
-Caption here
+Hello World!
```
Comment (kesslerb) [inline _posts/0000-01-07-kesslerb.md]: 🌵

**Comment by h9999** [comment]
approved

**Review by h9999 — APPROVED** [review]
reviewed approved

File: _posts/0000-01-07-kesslerb.md (hunk 1 above)
Comment (APBEI) [inline _posts/0000-01-07-kesslerb.md]: good

**Review by APBEI — APPROVED** [review]

**Review by kesslerb — COMMENTED** [review]
🌵 🌵

**Pull request merged**